_posts/guides/2021-12-21-guide_gamemode-info-surf.md

```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
 
 ## How Surf Works
 
-TODO
\ No newline at end of file
+You can surf by holding A on the right side of the ramp and D on the left without holding W.
+When getting on a ramp you need to look downwards, then slowly turn your camera until the bottom where you should be looking flat with the ramp.
+If you want to jump you need to slowly start looking at the ramp and when you're halfway up look directly at it.
```

Comment:
I'm a little confused by this, do you mean when starting a stage? If that's the case let's clarify with by starting with "To begin boarding the ramp by jumping". Though isn't this also the case if you don't jump?